OpenOversight/app/models.py

```diff
@@ -33,6 +33,7 @@ class Officer(db.Model):
     pd_id = db.Column(db.Integer, index=True, unique=False)
     assignments = db.relationship('Assignment', backref='officer', lazy='dynamic')
     face = db.relationship('Face', backref='officer', lazy='dynamic')
+    department_id = db.Column(db.Integer)
```

Comment:
do we not want to explicitly state these are foreign keys here and in the migration?

Reply:
also what happens if someone changes departments, say from UCPD to OPD?